packages/config/config/devices/0x014f/lb60z-1.json

```
@@ -23,39 +23,15 @@
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Initial Bulb Brightness",
-			"valueSize": 1,
-			"defaultValue": 0,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Full Brightness",
-					"value": 0
-				},
-				{
-					"label": "Prior Brightness Level",
-					"value": 1
-				}
-			]
+			"$import": "templates/config_template.json#init_brightness"
 		},
 		{
 			"#": "9",
-			"label": "Dim Step Level",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 99,
-			"defaultValue": 1,
-			"unsigned": true
+			"$import": "~/templates/master_template.json#dim_rate"
```

Shouldn't this be the dim step instead?
```suggestion
			"$import": "templates/config_template.json#dim_step_zwave"
```

---

In my mind, dim_rate is **how fast** and dim_step is **how much**